sdk_container/src/third_party/coreos-overlay/coreos/config/env/acct-group/incus

```diff
@@ -0,0 +1,8 @@
+cros_post_src_install_add_userdb_record(){
```

Comment:
Could you add comments what this hook is doing? Thanks. (Same for `acct-group/incus-admin` and `acct-group/lxc` hooks too, please.)